packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx

```diff
@@ -57,6 +39,25 @@ export const LatestRatingMovies = () => {
           .map((movie) => <MovieCard key={movie.id} movie={movie} />)
           .splice(2, 8)}
       </Masonry>
-    </Box>
+    </StyledMovieGrid>
   );
 };
+
+const StyledMovieGrid = styled(Box)`
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: linear-gradient(to right bottom, #003c2d, #00ffc2, #000000);
+`;
+
+const LatestRating = styled(Typography)`
+  height: 2.76rem;
+  font-size: 2.25rem;
+  font-weight: 700;
+  color: #ffffff;
+  text-align: center;
+  margin-bottom: 3rem;
+  margin-top: 3rem;
+  font-family: Inter;
```

Comment:
To be safe, I would suggest to define a fallback font in case for some reason Inter is not loaded, such as Arial, serif, sans-serif etc.

The way to do it is basically by listing the other fonts, comma separated. See https://www.w3schools.com/cssref/pr_font_font-family.php

Reply:
We have a fallback font in ThemeProvider, will it help in case Inter is not loaded?
![theme](https://github.com/HackYourFuture-CPH/CINEMANIA/assets/86012464/f9cb5079-fe24-4b36-add4-12afd1ab39a4)